### PR TITLE
Create to do list structure in webpack

### PR DESCRIPTION
**Pull request by Becky449**
##Set up a new project with webpack
##create to-do list 
##added js and styles

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+import './style.css';
+
+const listsContainer = document.querySelector('#tasks');
+const newListForm = document.querySelector('[data-new-list-form]');
+const newListInput = document.querySelector('[data-new-list-input]');
+
+const lists = [];
+
```
Comment (codepantha) [inline src/index.js]: - [ ] According to requirements, please create an array of some simple *to do tasks* (array of objects) with each object containing three keys - description \[`string`\], completed \[`bool`\], index: \[`number`\]

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRES CHANGES ♻️ 

## Hi @Becky449 👋, 

Good job so far! I can tell you've put in a lot of effort into this 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Descriptive PR
✔️ Used correct GitHub Flow
✔️ No linter errors
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (rloterh) [reply src/index.js]: well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Becky,  

Your project is complete! There is nothing else to say other than... it's time to merge it 🍾Congratulations!  🎉💯🌟 

## Highlights 👍
- Project is set up correctly
- Webpack is correctly configured

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**You can also consider:**
- I think you should change the header title _Authors_ to _Author_ in your ReadMe, as this project is worked on by a sole contributor and replace the sub heading text Author1 with your full name in your ReadMe.
- Live demo link in your ReadMe is broken, kindly endeavor to fix it.
Implementing the above change(s) will make your ReadMe appear more professional.
<br>
Keep soaring higher! 💻 🍷 🚀 
<br><br>
Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**